store/timer_layer.go

```diff
@@ -552,6 +552,22 @@ func (s *TimerLayerChannelStore) ClearCaches() {
 	return
 }
 
+func (s *TimerLayerChannelStore) CountPostsAfter(channelId string, timestamp int64, userId string) (int, *model.AppError) {
```

Comment:
Prob. better to have this change (and the others below) in separate PR than being tagged along to this one, since they don't seem to be related.